### DISCUSSION TITLE
Two questions about public link sharing

### DISCUSSION THREAD

**Discussion by chipoman** [ente-io/ente#3906] [Q&A] [answered]
Hello, please may I ask 2 questions about sharing a public link to an album?
When the recipient views my album, is the sort order that I have set maintained (oldest first)?
When the recipient views an individual photo from the album, will they see the photo description?
I have found the example, "Trip to Sikkim", which is sorted newest first and there are no photo descriptions but, of course, I don't know what the original album sort order was or whether photo descriptions had been added.
I currently have a free account, so cannot test this for myself.  I'm  considering moving from Flickr to the safer world of Ente (paid) later this month, when my Flickr sub runs out.

**Comment by vishnukvmd** [comment, marked as answer]
Both the sort order and the description will be available to the viewer.

**Reply by chipoman** [reply]
That's great to know, thanks @vishnukvmd!
